src/components/admin/user-accounts-tab.yml

```diff
@@ -27,7 +27,7 @@ userAccountsTab:
     en: user
     de: Benutzer
   role_accredited-author:
-    en: accredited author
+    en: accred. author
     de: Akkred. Autor
```

Comment:
Was just about to ask if the full name would fit now in German. But I guess it does not, in either language..

Reply:
Yes, otherwise the select has to wider again :-(